circuit/src/main/kotlin/com/slack/circuit/CircuitContent.kt

```diff
@@ -122,6 +122,8 @@ private fun CircuitContent(
 
   if (ui != null && presenter != null) {
     CircuitRender(presenter, ui)
+  } else if (circuit.onUnavailableContent != null) {
+    circuit.onUnavailableContent.invoke(screen)
```

Comment:
If you default this value in the parameter to `= circuit.onUnavailableContent` you can fold this

Reply:
Oh, yeah. Good idea 👍